manifests/pnp4nagios/config.pp

```diff
@@ -0,0 +1,77 @@
+class icinga2::pnp4nagios::config inherits icinga2::pnp4nagios {
+
+  $monitoring_type = 'icinga'
+  $log_type = 'syslog'
+  $debug_lvl = 1
+
+  file { "$httpd_path/$htpasswd_config":
+    ensure => $htpasswd_config_ensure,
+    owner  => $htpasswd_user,
+    group  => $htpasswd_group,
+    mode   => $htpasswd_config_mode,
+    content => template('icinga2/pnp4nagios.conf.erb'),
+    notify => Service['httpd'],
+  }
+
+  file { '/etc/nagios/passwd':
+    ensure  => file,
+    owner   => $htpasswd_user,
+    group   => $htpasswd_group,
+    notify  => Service['httpd'],
+  }
+
+  exec { 'htpasswd':
+    path    => '/bin:/usr/bin:/sbin:/usr/sbin',
+    command => "htpasswd -db /etc/nagios/passwd $nagios_web_user $nagios_web_pass",
+    subscribe   => File["/etc/nagios/passwd"],
+    refreshonly => true
+  }
+
+  file { '/etc/pnp4nagios/npcd.cfg':
+    owner => $htpasswd_user,
+    group => $htpasswd_group,
+    notify => Class[icinga2::pnp4nagios::service],
+    content => template('icinga2/npcd.cfg.erb'),
+  }
+
+  if $manage_php_timezone == true {
+    file_line{ 'date.timezone':
+      path => '/etc/php.ini',
+      line => "date.timezone = Europe/Amsterdam",
+      match => "^;date.timezone.*$",
```

Comment:
date.timezone = UTC     (It will work)
The only thing is, this parameter should not be blank.
Also, timezone is configurable via icinga web dashboard. Therefore date.timezone should be enabled with some timezone.